src/main/java/connectors/consumer_producer/TrainDataConsumer.java

```diff
@@ -0,0 +1,60 @@
+package connectors.consumer_producer;
+
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.serialization.LongDeserializer;
+
+import java.util.Collections;
+import java.util.Properties;
+
+public class TrainDataConsumer {
+    private static Consumer<Long, TrainData> createTrainDataConsumer() {
+        final Properties props = new Properties();
+        props.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG,
+                KafkaConstants.KAFKA_BROKERS);
+        props.put(ConsumerConfig.GROUP_ID_CONFIG,
+                KafkaConstants.GROUP_ID_CONFIG);
+        props.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG,
+                LongDeserializer.class.getName());
+        props.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG,
+                TrainDataDeserializer.class.getName());
+
+        // Create the consumer using props.
+        final Consumer<Long, TrainData> consumer =
+                new KafkaConsumer<>(props);
+
+        // Subscribe to the topic.
+        consumer.subscribe(Collections.singletonList(KafkaConstants.TOPIC_NAME));
```

Comment:
Perhaps we could add a comment here, does `subscribe` allow us to subscribe to multiple topics that are inside the given list? 

The current comment isn't really meaningful i think.